protocol/messenger_test.go

```diff
@@ -270,24 +270,23 @@ func (s *MessengerSuite) TestInit() {
 				key, err := crypto.GenerateKey()
 				s.Require().NoError(err)
 				contact := Contact{
-					ID:         types.EncodeHex(crypto.FromECDSAPub(&key.PublicKey)),
-					Name:       "Some Contact",
-					SystemTags: []string{contactAdded, contactBlocked},
+					ID:      types.EncodeHex(crypto.FromECDSAPub(&key.PublicKey)),
+					Name:    "Some Contact",
+					Blocked: true,
```

Comment:
If a contact is blocked would they also be added?

Reply:
yes